tests/unit/dedupe-tracked-test.js

```diff
@@ -142,4 +143,35 @@ module('Unit | Utils | @dedupeTracked', () => {
       'getter is called again after updating to a different value'
     );
   });
+
+  test('it works with an EmberObject', (assert) => {
```

Comment:
> `Should use `assert.expect()` when using assertions outside of the top-level test callback`

@boris-petrov FYI this error caused by arrow function. I've seen this before and it's enough to use `function` literal to make ESLint happy (haven't dug into root cause)

Reply:
Ah. It’s because of how QUnit does `this` binding: it cannot work with arrow functions.

Reply:
Sorry, my bad, I fixed the linter errors.